core-web/libs/dotcms-models/src/lib/dot-language.model.ts

```diff
@@ -1,6 +1,6 @@
 export interface DotAddLanguage {
```

Comment:
isnt called locale?

Reply:
right, the interface kept as it was, like the Database table, the `Locale` is visual.